suites/pacific/upgrades/tier-1_upgrade_test-4x-to-5x-container.yaml

```diff
@@ -165,3 +165,9 @@ tests:
     module: test_cephadm_shell_run.py
     desc: Verification cephadm shell when running from a non-monitor node
     polarion-id: CEPH-83575009
+
+- test:
+    name: test_check_non_exisiting_legacy_services
+    module: test_check_non_exisiting_legacy_services.py
```

Comment:
keep the module name user-friendly, keep it short and simple. may be `legacy_services.py`